### ISSUE TITLE
Adding custom sounds for chats is not accessible

### ISSUE THREAD

**Issue by Elshara** [UnigramDev/Unigram#2770]
**Describe the bug**
Telegram added support for this. I don't know how it works in the desktop client. On mobile it's not accessible.
**To Reproduce**
1. Go to 'Your recent chats'
2. Click on 'Someone's name, using the right click, applications key, or context menu'
3. Scroll down to 'Sounds and notifications, likely under mute'
4. There's no ability to add custom sounds for that user.

**Expected behavior**
I want to be able to know who messages me without having to tab around all over the place to respond to messages I add to folders deemed as important. The best way I can think of solving this issue, is somehow muting folders. But that doesn't work, so I have to use individual chats as a guidelines for important notifications only. Right now, it's one sound for groups and private messages. I want to update the behavior to have multiple sounds. Telegram desktop and mobile clients let you do this, by right clicking on a message that is displayed as music or voice, and yet Unigram has not got that option to add it to sounds.
**Screenshots**

**Version Info**
 - Unigram: [8.8.7433.0_x64]
 - Windows: [11 Version 22H2 build 25145.1000]

**Additional context**

**Comment by MarcAnt01** [comment]
@Elshara please fill the template, otherwise the issue will be closed

**Comment by Elshara** [comment]
Fixed.

**Comment by makisukurisu** [comment]
Sadly even if this was added to context menu it still wouldn't work at the moment.

Maybe resolving #2809 will add some sense for this change to happen. But AFAIK Windows doesn't allow you to change notification sound for Notification Center notifications.

**Comment by FrayxRulez** [comment]
Done
